core/storage/topic/gae_models.py

```diff
@@ -106,17 +106,17 @@ def _trusted_commit(
     def get_by_name(cls, topic_name):
         """Gets TopicModel by topic_name.
 
+        Note: For the query made, a list with one element is returned or
```

Comment:
This method, from the point of view of a caller, returns a single element or None. So you don't need to talk about lists here; it's kind of misleading.

Reply:
Done.